VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java

```diff
@@ -119,26 +110,26 @@ public static VoxelBrushManager initialize() {
     public void registerSniperBrush(Class<? extends IBrush> clazz, String... handles) {
         Preconditions.checkNotNull(clazz, "Cannot register null as a class.");
         for (String handle : handles) {
-            brushes.put(clazz, handle.toLowerCase());
+            brushes.put(handle.toLowerCase(), clazz);
             brushHandles.add(handle.toLowerCase());
         }
     }
 
     /**
      * Retrieve Brush class via handle Lookup.
      *
-     * @param handle Case insensitive brush handle
+     * @param handle Case-insensitive brush handle
      * @return Brush class
      */
     public Class<? extends IBrush> getBrushForHandle(String handle) {
         Preconditions.checkNotNull(handle, "Brushhandle can not be null.");
-        if (!brushes.containsValue(handle.toLowerCase())) {
+        if (!brushes.containsKey(handle.toLowerCase())) {
             return null;
         }
 
-        for (Map.Entry<Class<? extends IBrush>, String> entry : brushes.entries()) {
-            if (entry.getValue().equalsIgnoreCase(handle)) {
-                return entry.getKey();
+        for (String key : brushes.keySet()) {
+            if (key.equalsIgnoreCase(handle)) {
+                return brushes.get(key);
```

Comment:
here you can do 
```suggestion
       return brushes.get(handle.toLowerCase());
```
that could also get rid of the check on line 126 and the return null on 135

Reply:
there is no check to prevent uppercase handle registration (as far as i know). brush in brushes and the handle can be uppercase. So this is still needed?

Reply:
The method registerSniperBrush puts handle.toLowerCase in the map. And since the map is private we can be sure that there are no uppercase handles